src/WingetCreateCLI/Properties/Resources.resx

```diff
@@ -1400,4 +1400,10 @@ Warning: Using this argument may result in the token being logged. Consider an a
   <data name="ConfirmZippedBinary_Message" xml:space="preserve">
     <value>Does this executable depend on DLLs or any other files present in the zip archive?</value>
   </data>
+  <data name="SyncForkFailed_Message" xml:space="preserve">
+    <value>The forked repository could not be synced with the upstream commits. Sync your fork manually and try again.</value>
+  </data>
+  <data name="BranchMergeConflict_Message" xml:space="preserve">
+    <value>The forked repository could not be synced with the upstream commits due to a merge conflict.</value>

```

Comment:
Nit: Since the corresponding comment mentions that the user needs to resolve this manually. I suggest also mentioning that in the error message, which would also align nicely with the other message you've added 😊